src/bufr/DataObject/ArrayDataObject.h

```diff
@@ -24,9 +24,13 @@ namespace Ingester
         /// \param obsGroup Obsgroup were to add the variable
         /// \param name The name to associate with the variable (ex "latitude@MetaData")
         /// \param dimensions List of Variables to use as the dimensions for this new variable
+        /// \param chunks List of integers specifying the chunking dimensions
+        /// \param compressionLevel The GZip compression level to use, must be 1-6
```

Comment:
Double check this, I think it is actually 1-9 and 0 is an option too.

Reply:
Good catch. I think you are correct. I have not been able to find any documentation for the behavior for when you set the compression level to 0 however... I did run it that way and nothing bad happened at least :)